### ISSUE TITLE
Row indices did not export to excel

### ISSUE THREAD

**Issue by dshlai** [DeepSpace2/StyleFrame#30]
Hi! Thanks for contributing to StyleFrame. Please make sure to mention the Python, StyleFrame, pandas and openpyxl versions you are using.

styleframe: 1.2
pandas: 0.20.3
python: 2.7.11

Hi:

I am exporting a dataframe with labled row index:

```python
                          updated_at updated_by
code
Dragon_Base       2015-12-22 10:09:37   BPAdmin
Dragon_Cave       2015-12-22 10:09:37   BPAdmin
Officer_Hammer    2015-12-22 10:08:25   BPAdmin
Officer_Claw      2015-12-22 10:08:25   BPAdmin
```
However, the resulting excel didn't contain the row index as a column. 
Using pandas with openpyxl as writer did not have this behavior. Row index label was export as first column.

Thanks

**Comment by DeepSpace2** [comment]
Hi. In StyleFrame's `to_excel` the default is `index=False`. If you want the indexes in the excel you should pass `index=False` to `to_excel`.

**Comment by DeepSpace2** [comment]
Will be fixed in next version (1.6.1)

**Comment by DeepSpace2** [comment]
Fixed in version 1.6.1
